src/components/MyContactList.js

```diff
@@ -0,0 +1,21 @@
+import MyContactCard from './MyContactCard';
+import useStore from './Store';
+
+export default function MyContactList() {
+	const myContact = useStore(state => state.myContact);
+
+	return myContact.map(contact => {
+		return (
+			<MyContactCard
+				key={contact.id}
+				firstName={contact.firstName}
+				lastName={contact.lastName}
+				job={contact.job}
+				company={contact.company}
+				email={contact.email}
+				phone={contact.phone}
+				website={contact.website}
+			/>
```

Comment:
You could also pass it like this:

`<MyContactCard
	key={contact.id}
    contact={contact}
			/>`

Then you would receive it within you component as this:
`export default function MyContactCard({contact}) {`